### ISSUE TITLE
[USABILITY] Add better punctuation in help pop-up for color codes

### ISSUE THREAD

**Issue by joegrohens** [rokwire/safer-illinois-app#330]
**Describe the usability issue**
The list of conditions related to Orange color status is comma-separated. It's hard to parse for novice users who don't understand how the testing program works.

Each item in the list needs to be understood (by a user) as a separate and sufficient condition for Orange, not a combination of conditions.

Ideally this list would be a bullet list, but I get the impression that bullet formatting is hard to do in this pop-up.

Therefore, please use semicolon separator between each item in the list, as follows below.

**Revised text:**

Orange: First time user;  Past due for test;  Self-reported symptoms;  Received exposure notification; (or) Quarantined

**To Reproduce**
Go to Home > Your Health > Tap the circle i (i)
Help for status color definitions is displayed

**Screenshots**
![color help](https://user-images.githubusercontent.com/66988329/97189827-048d9e80-1773-11eb-8411-760c4ca71f80.jpeg)

**Additional context**
I am aware that Allie's more involved revisions for this pop-up are "on hold".
https://github.com/rokwire/safer-illinois-app/issues/242

The change I am asking for is not complicated, and will resolve frequently asked questions.

**Comment by pmarkhennessy** [comment]
@joegrohens We will wait until we redo the entire panel. This is lower priority at the moment.
